xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/core/wallet/SeedWalletGenerationResult.java

```diff
@@ -1,11 +1,15 @@
 package org.xrpl.xrpl4j.crypto.core.wallet;
 
 import org.immutables.value.Value;
+import org.xrpl.xrpl4j.crypto.core.keys.KeyPairService;
 import org.xrpl.xrpl4j.crypto.core.keys.Seed;
 
 /**
  * The result of generating a {@link Wallet} from a seed value.
+ *
+ * @deprecated Prefer {@link KeyPairService}.
  */
+@Deprecated
```

Comment:
If introduced in v3, let's remove instead of deprecate

Reply:
Fixed in 3c3459f9491f094750d0fcb910653db452eedcfb